### ISSUE TITLE
[Feature Request] Add option to filter automatically generated subtitles

### ISSUE THREAD

**Issue by rdaraujo** [yuliskov/SmartTube#1993]
It seems Google has added “automatically generated” subtitles for every language they know in every video.

It would be great to have an option to filter these subtitles from the list, showing only the official ones from the video submitter.

**Comment by hiroo916** [comment]
Agree.   And to expand on this request, I suggest adding a preference section in the settings to select the languages to display:

X  Show auto-generated subtitles
X  Only show these subtitle languages:
     X  English
     X  Japanese 
     ... etc

Right now there are so many languages that it takes a while to scroll down to "English" which is likely the majority selection for available subs.

**Comment by gagalgang** [comment]
I desperately need this function.

There are so many languages I don't use.
